Nodejs/Product/Nodejs/Workspace/WorkspaceExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.NodejsTools.TypeScript;
+using Microsoft.VisualStudio.Workspace;
+
+namespace Microsoft.NodejsTools.Workspace
+{
+    internal static class WorkspaceExtensions
+    {
+        public static async Task<(bool, TsConfigJson)> IsContainedByTsConfig(this IWorkspace workspace, string filePath)
```

Comment:
>(bool, TsConfigJson) [](start = 33, length = 20)

Is the `(true, null)` case interesting?  If not, consider just returning the `TsConfigJson`.